shared/README.md

```diff
@@ -7,25 +7,57 @@
 - Service Principal
```

Comment:
Service principal is not required here unless you're setting this up for CI which is not in scope.  Please remove.  When we get to adding CI, we will provide specific instructions that require this.